Presentation.Web/Models/API/V2/Response/Generic/Validity/ValidityResponseDTO.cs

```diff
@@ -5,10 +5,14 @@ namespace Presentation.Web.Models.API.V2.Response.Generic.Validity
     public class ValidityResponseDTO
     {
         /// <summary>
-        /// Determines if the entity is considered valid. This is computed from both "EnforcedValid" as well as ValidFrom/To
+        /// Determines if the entity is considered valid. This is computed from both "EnforcedValid" as well as ValidFrom/To. In case of the ItSystemUsage it's computed using CheckSystemValidity method
         /// </summary>
         public bool Valid { get; set; }
         /// <summary>
+        /// Determines if the entity is considered valid, computed by IsActiveAccordingToDateFields property
```

Comment:
No... this is api v2. The terms that exist are the ValidFrom and ValidTo properties.. right?.. So

`Determines if the entity is considered valid based on the validity period defined by ValidFrom and ValidTo`